### ISSUE TITLE
dealing with blanks in the logsheets

### ISSUE THREAD

**Issue by kmexter** [emo-bon/data-quality-control-action#4]
For rows that contain blank replicate types, a number of changes are needed
1) Instead of calling them blank1,2 when repeated, call them just blank but have a _1 and _2 in the source mat id. Need to figure out how to do this - see https://github.com/emo-bon/observatory-hcmr-1-crate/issues/13
2) the only relevant information from the sampling tab for blanks are the following:
scientific name --> it's mandatory, so yes. Now it is "unidentified" but I think this is wrong. I added an [issue](https://github.com/emo-bon/sequencing-profile/issues/2) about it. 
tidal stage --> optional, so no.  Also, not relevant. 
depth --> mandatory, so yes. For now, I have used 0 (there has to be a number).
sampl collect device --> optional, so no. Also, not relevant. 
samp mat process and deviations --> optional, but I would say yes.
membr cut --> optional, but I would say yes.
size frac low and up --> optional, but I would say yes.
time fi --> optional, so no. Also, not relevant. 
env_material  --> it's mandatory, so yes.  Right now, for the sediment blanks it is "swab" for the water blanks it is "Milli-Q water" (based on the SOPs).
3) no measurements should be added for these samples

We will need a separate function to deal with blanks for the transformed logsheets OR just for the ttl template. To be taken up after my summer break

**Comment by kmexter** [comment]
The work to do this should be done via https://github.com/emo-bon/observatory-hcmr-1-crate/issues/13 and the solution proposed there can be adopted. For point 2, I will take this up with @laurianvm first and then maybe Bram

**Comment by kmexter** [comment]
Has no effect on the QC etc but point 2 bring up with @laurianvm

**Comment by laurianvm** [comment]
@kmexter is this GH issue still relevant given changes that have occured since its creation?

**Comment by kmexter** [comment]
I don't think so - the blanks are now called blank_1 etc in the logsheets, but I suspect that the error-catching that the QC workflow does will need updating for that. @marc?

**Comment by laurianvm** [comment]
blanks are handled correctly in the logsheets: 
when a sample is typed as blank, its has no values in measurements.csv (rather 'NA')

2 instances where blank samples do have measurements have been identified (this appear to be due copy-pasting) and are being handled:
- https://github.com/emo-bon/observatory-aaot-crate/blob/main/logsheets/transformed/water_measured.csv#L81
- https://github.com/emo-bon/observatory-vb-crate/blob/main/logsheets/transformed/water_measured.csv

**Comment by kmexter** [comment]
grand, so the final question @isanti is whether you want to have as part of the QC a notification that a blank row has NOT got NA in certain columns, or not?
